### PR TITLE
Dashboard Brake Bias, Fault Page Indicator, TV Driver Page

### PR DESCRIPTION
**Pull request by mcgalliard**

### CODE REVIEW

**Review by LukeOxley — APPROVED** [review]
lgtm

File: source/dashboard/lcd/lcd.h [hunk 1]
```diff
@@ -168,6 +167,10 @@ typedef struct {
 
 } tv_settings_t;
 
+typedef struct {
+  uint16_t brake_bias_adj;  // 0 to 10000 for the page
```
Comment (AdityaAsGithub) [inline source/dashboard/lcd/lcd.h:171]: If there are other variables similar to this that need to be stored we should add it to this struct to keep everything organized.

**Review by AdityaAsGithub — CHANGES_REQUESTED** [review]
some nitpicky stuff. Like it overall

File: source/dashboard/lcd/lcd.c [hunk 2]
```diff
@@ -1000,10 +956,27 @@ void update_data_pages() {
...
-                set_text(TV_FR, NXT_TEXT, "S");
-                set_text(TV_LR, NXT_TEXT, "S");
-                set_text(TV_RR, NXT_TEXT, "S");
+                // Value MUST be between 0 and 9999 and represents the percentage
+                // We will do the division as a float and then "convert" to an integer 
+                set_value(BRAKE_BIAS_FLT, NXT_VALUE, race_page_data.brake_bias_adj);
+                // Grab signal?
+                if (can_data.throttle_vcu.stale)
+                {
+                    set_value(TV_RL_FLT, NXT_VALUE, 0);
```
[the remark this answers is not in the document]
Comment (mcgalliard) [reply source/dashboard/lcd/lcd.c:965]: kk

File: source/dashboard/lcd/lcd.c [hunk 3]
```diff
@@ -1000,10 +956,27 @@ void update_data_pages() {
...
+                    set_value(TV_RR_FLT, NXT_VALUE, 0);
+                }
+                else
+                {
+                    int adj_vcu_rl = can_data.throttle_vcu.vcu_k_rl * 100 * 100;
```
[the remark this answers is not in the document]
Comment (mcgalliard) [reply source/dashboard/lcd/lcd.c:970]: Display expects 0 - 10000 not 0 - 1, one is conversion to percent the other is conversion to scale, I can add #defines like the rest of them

File: source/dashboard/lcd/lcd.h [hunk 4]
```diff
@@ -100,12 +100,11 @@
 //Race/Data pages
 #define THROT_BAR "j1"
 #define POW_LIM_BAR "j0"
+#define BRAKE_BIAS_FLT "x0"
+#define TV_RL_FLT "x1"
+#define TV_RR_FLT "x2"
```
[the remark this answers is not in the document]
Comment (mcgalliard) [reply source/dashboard/lcd/lcd.h:105]: I like the floats, I did some research over spring break and they've been working, unless you feel strongly against I'd like to keep em.

File: source/dashboard/lcd/lcd.h (hunk 1 above)
Comment (mcgalliard) [reply source/dashboard/lcd/lcd.h:171]: Will explore

Comment (mcgalliard) [reply source/dashboard/lcd/lcd.h:171]: Explored: other items on race page for now come directly from can signals, so not necessary to add to struct yet

File: source/dashboard/lcd/lcd.c (hunk 2 above)
[the remark this answers is not in the document]
Comment (mcgalliard) [reply source/dashboard/lcd/lcd.c:965]: Setting to 77.77 for now

**Review by AdityaAsGithub — APPROVED** [review]

**Pull request merged**